### PR TITLE
Add aria label to Header Component

### PR DESCRIPTION
**Pull request by PaulChinkwende**
Added aria label of BBC News to our headline.

* [ ] Fixes https://github.com/bbc/simorgh/issues/345
* ~[ ] Tests added for new features~

* [ ] Test engineer approval

### CODE REVIEW

**Review by radiocontrolled — DISMISSED** [review]
LGTM 👍

File: src/app/components/Article/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -10,6 +10,7 @@ Array [
 }
 
 <header
+    aria-label="BBC"
```
Comment (pjlee11) [inline src/app/components/Article/__snapshots__/index.test.jsx.snap]: As per the issue this should be 'BBC News'

**Review by pjlee11 — APPROVED** [review]

**Review by radiocontrolled — APPROVED** [review]
Updated aria-label is approved.

**Comment by pjlee11** [comment]
The issue description was slightly misleading in that the offscreen text it mentions is actually being handled as part of #389 therefore the aria role added in this PR isn't needed because the text is going to be added here - https://github.com/bbc/simorgh/pull/389/files#diff-7895fecf86d2ee4b69c3688459a819ccR4

**Pull request merged**